phpunit.xml.dist

```diff
@@ -9,7 +9,7 @@
          processIsolation="false"
          stopOnFailure="false"
          syntaxCheck="false"
-         bootstrap="Tests/bootstrap.php"
+         bootstrap="tests/bootstrap.php"
```

Comment:
We manage libs that are not bundle and follow this directory structure: https://github.com/sonata-project/GoogleAuthenticator/ . So it should be fine or at least, possible.